test/mirror_vfd.c

```diff
@@ -2609,6 +2610,7 @@ main(void)
     if (nerrors == 0) {
         nerrors -= test_fapl_configuration();
         nerrors -= test_xmit_encode_decode();
+        nerrors -= test_on_disk_zoo();
```

Comment:
Looks like test_on_disk_zoo is already tested on line 2615 originally. Should remove this.